redirects.json

```diff
@@ -116,6 +116,10 @@
       "source": "/:locale/docs/guides/debugging-getting-started",
       "destination": "/:locale/learn/getting-started/debugging"
     },
+    {
+      "source": "/:locale/docs/guides/abi-stability/",
```

Comment:
FYI this redirect is wrong, should be `"source": "/:locale/guides/abi-stability/",`